src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpRawlsDAO.scala

```diff
@@ -81,6 +81,12 @@ class HttpRawlsDAO( implicit val system: ActorSystem, implicit val executionCont
   override def getWorkspace(ns: String, name: String)(implicit userToken: WithAccessToken): Future[WorkspaceResponse] =
     authedRequestToObject[WorkspaceResponse]( Get(getWorkspaceUrl(ns, name)) )
 
+  override def accessCheck(workspaceIds: Seq[String])(implicit userToken: WithAccessToken): Future[Seq[WorkspacePermissionsPair]] = {
+    implicit val WorkspaceAccessLevelFormat = WorkspaceACLJsonSupport.WorkspaceAccessLevelFormat
+    implicit val WorkspacePermissionsPairFormat = jsonFormat2(WorkspacePermissionsPair)
+    authedRequestToObject[Seq[WorkspacePermissionsPair]]( Post(workspaceAccessCheckUrl, workspaceIds) )
+  }
```

Comment:
once the rawls PR merges and we can get the latest version of the model jar, the implicits here should be able to go away.